src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -147,14 +147,29 @@ public SyntaxNode FieldDeclaration(IFieldSymbol field, SyntaxNode? initializer)
         /// <summary>
         /// Creates a method declaration.
         /// </summary>
-        public abstract SyntaxNode MethodDeclaration(
+#pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
```

Comment:
😕 Why is this showing up now? I see another overload, but it doesn't have optional parameters.

Reply:
no clue.  i have no idea how this analyzer works. I've *never* found it valuable.  It's far too aggressive and actively gets the rules wrong.  So every time it comes up, it just gets suppressed.